configs/vscode.json

```diff
@@ -22,5 +22,7 @@
   "explorer.confirmDelete": false,
   "extensions.ignoreRecommendations": true,
   "workbench.colorTheme": "Tokyo Night",
-  "window.titleBarStyle": "custom"
+  "window.titleBarStyle": "custom",
+  "files.insertFinalNewline": true,
+  "editor.scrollBeyondLastLine": false
```

Comment:
while I'm all for insertFinalNewLine, scrollBeyongLastLine is not necessary IMO

Reply:
Fair enough 👍🏻